packages/dds/merge-tree/src/test/obliterate.rangeExpansion.spec.ts

```diff
@@ -0,0 +1,383 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+import { ReconnectTestHelper } from "./reconnectHelper.js";
+import { TestClient } from "./testClient.js";
+
+describe("obliterate", () => {
+	let client: TestClient;
+	let events: number[];
+
+	beforeEach(() => {
+		client = new TestClient({
+			mergeTreeEnableObliterate: true,
+		});
+		events = [];
+		client.startOrUpdateCollaboration("local");
+		for (const char of "hello world") {
+			client.applyMsg(
+				client.makeOpMessage(
+					client.insertTextLocal(client.getLength(), char),
+					client.getCurrentSeq() + 1,
+				),
+			);
+		}
+		assert.equal(client.getText(), "hello world");
+	});
+
+	it("obliterate adjacent insert", () => {
+		const helper = new ReconnectTestHelper();
+
+		helper.insertText("A", 0, "|ABC>");
+		helper.processAllOps();
+		helper.obliterateRange("A", { pos: 0, side: 1 }, { pos: 4, side: 1 });
```

Comment:
I can never remember whether 1 is `Side.Before` or `Side.After`. Can we use the enum instead of magic numbers for clarity?